aten/src/ATen/native/TensorConversions.cpp

```diff
@@ -117,4 +117,19 @@ Tensor to_mkldnn_backward(const Tensor& grad, const Tensor& input_) {
   return grad.to_dense();
 }
 
+Tensor view_dtype(const Tensor& self, ScalarType dtype) {
+  if (self.scalar_type() == dtype) {
+    return self;
+  }
+  TORCH_CHECK(self.element_size() == c10::scalarTypeToTypeMeta(dtype).itemsize(),
```

Comment:
nit: you can reuse the `scalarTypeToTypeMeta` result below.

Reply:
fixed